### DISCUSSION TITLE
No module named 'ark'

### DISCUSSION THREAD

**Discussion by c0urtneyt** [angelolab/toffy#298] [Q&A]
Hello,
I updated toffy, and after I started getting this error (attached) saying that there is no module named 'ark'. I  have tried to pip install ark as well and it still does not work. The ark-analysis folder is also present within the toffy folder. 
<img width="844" alt="toffy_ark" src="https://user-images.githubusercontent.com/40770583/203607351-1f6bfc6c-816b-4a83-9a64-fb5b1cf0fd42.PNG">
<img width="1002" alt="ark_error" src="https://user-images.githubusercontent.com/40770583/203607365-a307419f-69b6-4111-8441-a78d44164911.PNG">

**Comment by srivarra** [comment]
Hi @c0urtneyt, we recently fixed the issue you are having here in #295. You should be able to see the these changes by updating to the latest changes from main.

In your terminal, change directory into the `toffy` repo. Then run `git pull` when you are on the `main` branch.
